assets/sass/components/global/_googlesitekit-use-snippet-switch.scss

```diff
@@ -0,0 +1,36 @@
+/**
+ * UseSnippetSwitch styles.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
```

Comment:
Ah, this old copy/paste chestnut :)
```suggestion
 * Site Kit by Google, Copyright 2022 Google LLC
```

Reply:
Only if we could use current_year in the comments.

Reply:
Hahah, yes that would be handy. I keep meaning to setup a VSCode snippet for this, which could insert the current year along with the rest of the header...